vimrcs/basic.vim

```diff
@@ -152,11 +149,28 @@ if has("gui_running")
     set guioptions-=e
     set t_Co=256
     set guitablabel=%M\ %t
+    " Set bigger font since we usually have big screen now
+    if has('gui_win32')
+      set guifont=DejaVu\ Sans\ Mono:h12
+      " For non-Latin languages, choose available font in 
+      "    your system and set guifontwide below
+      set guifontwide=Microsoft\ Yahei\ Mono:h11
```

Comment:
set guifontwide is aimed to display wide characters like CJK (Chiese Japanese Korean)